src/Microsoft.Performance.Toolkit.Engine/EngineExecutionResults.cs

```diff
@@ -20,6 +24,7 @@ public sealed class RuntimeExecutionResults
         private readonly IDataExtensionRetrievalFactory retrievalFactory;
         private readonly IDataExtensionRepository repository;
         private readonly IEnumerable<DataCookerPath> sourceCookers;
+        private readonly IDictionary<TableDescriptor, ICustomDataProcessor> tableToProcessorMap;
```

Comment:
>ICustomDataProcessor [](start = 54, length = 20)

is it possible for this Table to exist in more than one instance of the same ICustomDataProcessor?

Reply:
An engine is a single session, so this shouldn't be an issue.

---
In reply to: [592042897](https://github.com/microsoft/microsoft-performance-toolkit-sdk/pull/56#discussion_r592042897) [](ancestors = 592042897)

Reply:
nope :)